src/pudl/metadata/resources/eiaaeo.py

```diff
@@ -0,0 +1,514 @@
+                "report_year",
+                "electricity_market_module_region",
+                "aeo_case",
+                "projection_date",
+                "aeo_service_category",
+                "real_value_base_year",
+            ],
+        },
+        "sources": ["eiaaeo"],
+        "field_namespace": "eiaaeo",
+        "etl_group": "eiaaeo",
+    },
+    "core_eiaaeo__fuel_consumption_by_type": {
+        "description": (
+            "Projected fuel consumption by the electric power sector, "
+            "including electricity-only and combined-heat-and-power plants "
+            "that have a regulatory status."
+        ),
+        "schema": {
+            "fields": [
+                "report_year",
+                "electricity_market_module_region",
+                "aeo_case",
+                "projection_date",
+                "aeo_fuel_consumption_type",
+                "fuel_consumed_mmbtu",
+            ],
+            "primary_key": [
+                "report_year",
+                "electricity_market_module_region",
+                "aeo_case",
+                "projection_date",
+                "aeo_fuel_consumption_type",
+            ],
+        },
+        "sources": ["eiaaeo"],
+        "field_namespace": "eiaaeo",
+        "etl_group": "eiaaeo",
+    },
+    "core_eiaaeo__electric_sector_fuel_cost_by_type": {
+        "description": (
+            "Projected fuel prices to the electric power sector, including "
+            "electricity-only and combined-heat-and-power plants that have a "
+            "regulatory status."
+        ),
+        "schema": {
+            "fields": [
+                "report_year",
+                "electricity_market_module_region",
+                "aeo_case",
+                "projection_date",
+                "aeo_fuel_cost_type",
+                "real_value_base_year",
+                "fuel_cost_mmbtu",  # this is *nominal* dollars, but we probably want to standardize to some sort of constant-dollar value?
+            ],
+            "primary_key": [
+                "report_year",
+                "electricity_market_module_region",
+                "aeo_case",
+                "projection_date",
+                "aeo_fuel_cost_type",
+                "real_value_base_year",
+            ],
+        },
+        "sources": ["eiaaeo"],
+        "field_namespace": "eiaaeo",
+        "etl_group": "eiaaeo",
+    },
+    "core_eiaaeo__electric_power_sector_emissions": {
+        "description": (
+            "Projected emissions from the electric power sector, including "
+            "electricity-only and combined-heat-and-power plants that have a "
+            "regulatory status."
+        ),
+        "schema": {
+            "fields": [
+                "report_year",
+                "electricity_market_module_region",
+                "aeo_case",
+                "projection_date",
+                "carbon_mass_tons",
+                "co2_mass_tons",
```

Comment:
I chose to unstack this because I already saw a bunch of these fields lying around. :shrug: